hy/contrib/hy_repr.hy

```diff
@@ -82,10 +82,10 @@
     (+ (get syntax (first x)) (hy-repr (second x)))
     (+ "(" (-cat x) ")"))))
 
-(hy-repr-register HySymbol str)
-(hy-repr-register [str-type bytes-type HyKeyword] (fn [x]
-  (if (and (instance? str-type x) (.startswith x HyKeyword.PREFIX))
-    (return (cut x 1)))
+(hy-repr-register [HySymbol HyKeyword] str)
+(hy-repr-register [str-type bytes-type] (fn [x]
+  (if (and (instance? str-type x) (.startswith x ":"))
+      (return x))
```

Comment:
I don't think this is right, since `:foo` doesn't evaluate to the string ":foo"; it now produces a HyKeyword. This ought to be causing a test failure, so I guess I screwed up the test for this.

Reply:
Good catch

Reply:
Yeah, I left that in because of this assert:

```clj
(assert (= (hy-repr (str ':mykeyword)) ":mykeyword"))
```

But I guess that doesn't make sense, we're breaking that.